README.md

```diff
@@ -44,6 +67,12 @@ You can also use the CLI by executing the following command from your container:
 $ docker-run --rm fundocker/learninglocker:v2.6.2 node cli/dist/server
 ```
 
+### Use xapi-service image
```

Comment:
```suggestion
### Using the xapi-service image
```

Reply:
I should also change the title for learning locker ? In one hand we have `Using the xapi-service image` and in the other hand `Use learning locker image`

Reply:
I thought I've added it.